client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -137,13 +136,8 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
     const axiosInterceptorId = useSelector<StoreStateType, number>(state => state.investigation.axiosInterceptorId);
     const windowTabsBroadcastChannel = useRef(new BroadcastChannel(BC_TABS_NAME));
 
-    const generateStatusFilterBySelectedStatues = (statuses: InvestigationMainStatus[]) => {
-        return statuses.includes(allStatusesOption)
-            ? [] : statuses;
-    };
```

Comment:
why removing ```allStatusesOption```?

Reply:
To be consistent with the desk filter and it's redundant because you can clear the filter autocomplete to show all the statuses.